src/gtpu/encap.c

```diff
@@ -707,8 +738,9 @@ int update_urr_counter_and_send_report(struct pdr *pdr, struct far *far, u64 vol
             // TODO: FAR ID for Quota Action IE for indicating the action while no quota is granted
             if (triggers[i] == USAR_TRIGGER_START){
                 ret = DONT_SEND_UL_PACKET;
-            }                 
-            convert_urr_to_report(urrs[i], &report[i]);
+            } 
+            urr_counter = get_urr_counter_by_trigger(urrs[i], triggers[i]);                
+            convert_urr_to_report(urr, urr_counter, &report[i]);
```

Comment:
@chen042531 @tim-ywliu 
I think the first parameter should be `urrs[i]` not `urr`, this change introduce a new bug for online charging feature.
I propose a fix for this issue in #143 , please help to review the PR.

Thanks!